README.md

```diff
@@ -148,6 +148,7 @@ To use, just run: `brotli subFile.ass` and use the .br result file with the subU
 * python-html5lib
 * python-chardet
 * gperf
+* python3 ply (`pip3 install ply`)
```

Comment:
Same `python3-*` change here.

Not required, but ideally, the dependencies would be split in deps needed only for emscripten and deps otherwise needed to build JSO. This would allow to skip superfluous manual installs when using system emscripten.